test/jdk/java/lang/System/PropertyTest.java

```diff
@@ -80,6 +80,7 @@
                 {"user.dir"},
                 {"java.runtime.version"},
                 {"java.runtime.name"},
+                {"native.encoding"},
```

Comment:
Does this test differentiate between read-only and modifiable properties? "native.encoding" looks like it's explicitly overridable. Or do we need a test to verify it's not overridable?